.ci/bl-specific.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 
-echo "This is a placeholder script for potential future updates"
+sudo mkdir -p /etc/bluesky
+sudo touch /etc/bluesky/kafka.yml
```

Comment:
```suggestion
sudo touch /etc/bluesky/kafka.yml

TODO: add a realistic config file so that `nslsii.configure_base(..., publish_documents_with_kafka=True)` does not fail.
```

Reply:
Oops, forgot the hash symbol to make it a proper comment. Will submit a PR with the fix.